src/types/tema.ts

```diff
@@ -63,3 +63,20 @@ export enum Tema {
     YRA = 'Yrkesrettet attføring',
     YRK = 'Yrkesskade / Menerstatning'
 }
+
+export type TemaKey = keyof typeof Tema;
+
+const TEMA_LIST = Object.entries(Tema);
+export const getYtelseByTema = (temaKey: TemaKey | string | null): Tema | null => {
+    if (temaKey === null) {
+        return null;
+    }
+    const found = TEMA_LIST.find(([k]) => k === temaKey);
+    return typeof found === 'undefined' ? null : found[1];
```

Comment:
Legges det noe annet i `found[0]` her?

Reply:
Ja, `found` er et array som alltid har 2 verdier; `[string, Tema]`.
Skulle gjerne brukt destructuring her for å gjøre det tydelig, men `Array.find()` returnerer enten typen i arrayet eller `undefined` (som ikke kan desctructes).

Dette er noe jeg så at ble litt utydelig selv. Vurderte denne løsning mot å ev. finne en løsning som mest sannsynlig krever mer kode og tid, men som til gjengjeld kan være tydeligere.

Nå har du bekreftet det jeg fryktet; at dette ikke er tydelig nok og mest sannsynlig vil forvirre oss i fremtiden.
Jeg ser om jeg finner en annen løsning.

Reply:
Alternativ løsning er pushet. Selv synes jeg den er mer forståelig, men jeg har jo skrevet den.